docs/user-guide/troubleshooting.rst

```diff
@@ -85,6 +85,20 @@ processes to write to the same file path, and so Pulp had to do something differ
 widely used logging protocol, and given the distributed nature of Pulp it was the most appropriate
 logging solution available.
 
+Task ID
+^^^^^^^
+
+Since Pulp is a multi-threaded application, Pulp will attempt to log a shortened 8 character version
```

Comment:
Perhaps `multi-process application` would be more relevant since the subject is tasking.